### DISCUSSION TITLE
Chia wallet Network

### DISCUSSION THREAD

**Discussion by Sakuya-CN** [Chia-Network/chia-blockchain#9725] [Support]
Why did my chia wallet change from the main network to testnet7? My XCH coin has become txch. How to solve this problem

![1641263028(1)](https://user-images.githubusercontent.com/56291932/148001268-e52e4ca0-d371-46dd-ba88-9844a1ef424e.png)

**Comment by emlowe** [comment]
That is set in config.yaml - typically set by using the `chia configure --testnet true` command and can be undone with `chia configure --testnet false`

**Reply by Jacek-ghub** [reply]
Do changes done with "chia configure --change" are picked up immediately, or chia restart is still needed (for those settings that are normally read only at the startup time)?

**Reply by emlowe** [reply]
You need to restart all the services. (so if using the UI, you need to close and re-open the UI)

**Reply by Sakuya-CN** [reply]
> 这是在 config.yaml 中设置的 - 通常使用`chia configure --testnet true`命令设置，并且可以撤消`chia configure --testnet false`

Oh, thank you for solving the big problem for me. Thank you very much
